README.md

```diff
@@ -330,7 +330,7 @@ createElement(document, "button", { namespace: "xul" }); // manually set namespa
 
 ### About Zotero API
 
-Zotero docs are outdated and incomplete. Clone <https://github.com/zotero/zotero> and search the keyword globally.
+Zotero docs are outdated and incomplete. Clone [Zotero Code Repo](https://github.com/zotero/zotero) and search the keyword globally.
```

Comment:
we'd better keep the raw link here so that people can directly copy the link and clone

Reply:
Ok, I'll roll back these changes and take care of these issues later.